gulpfile.js

```diff
@@ -13,3 +13,5 @@ exports.dev = parallel(
   library.dev
 );
 exports.extractTranslations = extractTranslations;
+
+exports.test = templates.test;
```

Comment:
🧪

Reply:
I thought we might want to leave this in here because it's useful for testing. Thoughts?

Reply:
Removing